omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_8/ConceptSearchHandler1_8.java

```diff
@@ -73,8 +77,11 @@ public PageableResult search(RequestContext context) throws ResponseException {
 		String name = context.getParameter("name");
 		String conceptClass = context.getParameter("class");
 		String searchType = context.getParameter("searchType");
+		String uuid = context.getParameter("uuid");
+		String reference = context.getParameter("reference");
```

Comment:
What is the use of the uuid and reference parameters?

Reply:
Am using them to support retrieving concepts by both uuid and SAME-AS mapping.

Reply:
Both of the parameters(uuid and reference) should be used when searching for concepts.

Reply:
Isn't the current end point already fetching by both uuid and mapping?

Reply:
I can only see fetching by [mapping ](https://github.com/openmrs/openmrs-module-webservices.rest/blob/42aa1d2f31283ea439c0cc887ba9f64f88f9a492/omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_8/ConceptSearchHandler1_8.java#L142).

Reply:
Did you test the end point and confirm that fetching by uuid does not work?

Reply:
Have you actually tested the end point in your browser or any tools like curl, postman, etc and confirmed that the existing end point cannot fetch a concept by uuid and mapping?

Reply:
> Have you actually tested the end point in your browser or any tools like curl, postman, etc and confirmed that the existing end point cannot fetch a concept by uuid and mapping?

I hadn't done this. Am doing it out to confirm preferably using postman

Reply:
When you pass the uuid and mapping the endpoint can fetch(GET) a concept.

Reply:
Thoughtfully the ticket  requirement is to support returning a list of(many) concepts from a single search.

Reply:
> Then what did you mean by this? `What we do not have is a functionality that can fetch a concept by both uuid and mapping.`

I hadn't carried out the testing. So we do have the functionality.